code/backend/batch/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -1,12 +1,9 @@
 from typing import Optional
 from datetime import datetime, timedelta
-from azure.storage.blob import (
```

Comment:
Most of the tests that are now in the root will have to have their import changed to reflect this folder move. This is also the reason for the failing build. (Adding `.backend.batch` should fix it!)

Reply:
Thanks! I ignored here. :)